menuconfig.py

```diff
@@ -270,17 +389,10 @@ def _init_styles():
 #
 # Obscure Python: We never pass a value for color_attribs, and it keeps
 # pointing to the same dict. This avoids a global.
-def _style(fg_color, bg_color, attribs, no_color_extra_attribs=0,
-           color_attribs={}):
+def _style_attr(fg_color, bg_color, attribs, color_attribs={}):
     # Returns an attribute with the specified foreground and background color
     # and the attributes in 'attribs'. Reuses color pairs already created if
     # possible, and creates a new color pair otherwise.
-    #
-    # Returns 'attribs | no_color_extra_attribs' if colors aren't supported.
-
-    if not curses.has_colors():
-        return attribs | no_color_extra_attribs
```

Comment:
This is still needed, because `curses.init_pair()` throws an exception on terminals that don't support color.

A good test is to turn with `TERM=vt100`.

Reply:
(Won't run into many terminals that don't support color these days, but sometimes wonky stuff can happen with people's TERM settings, and it's nice to not crash then.)

Reply:
That was actually a derp on my side ... doing too many things at once isn't always a good idea :)

Fixed.